src/StdUtils.sol

```diff
@@ -4,12 +4,12 @@ pragma solidity >=0.6.2 <0.9.0;
 import "./console2.sol";
 
 abstract contract StdUtils {
+    uint256 private constant INT256_MIN_ABS =
+        57896044618658097711785492504343953926634992332820282019728792003956564819968;
```

Comment:
_(arbitrary comment location to start a thread)_

Looks like there was also a test failure in CI:

```
[FAIL. Reason: Arithmetic over/underflow Counterexample: calldata=0x87b338ab000000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000018000000000000000000000000000000000000000000000000000000000000000, args=[0, -57896044618658097711785492504343953926634992332820282019728792003956564819967, -57896044618658097711785492504343953926634992332820282019728792003956564819968]] testBoundInt(int256,int256,int256) (runs: 630, μ: 7029, ~: 7054)
Traces:
  [2013] StdUtilsTest::testBoundInt(0, -57896044618658097711785492504343953926634992332820282019728792003956564819967, -57896044618658097711785492504343953926634992332820282019728792003956564819968) 
    └─ ← "Arithmetic over/underflow"
```

Reply:
[c43e95b](https://github.com/foundry-rs/forge-std/pull/253/commits/c43e95be0f0771d0d188cd1b855ad0fe972b5b5c)